pkg/jobrunaggregator/jobrunaggregatorlib/jobrun_locator_release_jobs.go

```diff
@@ -9,12 +9,19 @@ import (
 	prowjobv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
 )
 
-func GetPayloadTagFromProwJob(prowJob *prowjobv1.ProwJob) string {
+// GetPayloadTagFromProwJobForAggregatedJob gets the payload tag from prow jobs for aggregated jobs.
+func GetPayloadTagFromProwJobForAggregatedJob(prowJob *prowjobv1.ProwJob) string {
 	return prowJob.Labels["release.openshift.io/analysis"]
 }
 
+// GetPayloadTagFromProwJobForNonAggregatedJob gets the payload tag from prow jobs for non-aggregated jobs.
+func GetPayloadTagFromProwJobForNonAggregatedJob(prowJob *prowjobv1.ProwJob) string {
+	return prowJob.Annotations["release.openshift.io/tag"]
+}
+
 func NewPayloadAnalysisJobLocatorForReleaseController(
 	jobName, payloadTag string,
+	isAggregated bool,
```

Comment:
this results in unreadable function calls. Create two methods or collapse the cases

Reply:
Done